osu.Game/Overlays/Chat/ChannelControl/ControlItemMention.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+
+namespace osu.Game.Overlays.Chat.ChannelControl
+{
+    public class ControlItemMention : CircularContainer
+    {
+        private int mentionCount = 0;
+        public int MentionCount
+        {
+            get => mentionCount;
+            set
+            {
+                if (value == mentionCount)
+                    return;
+
+                mentionCount = value;
+                updateText();
+            }
+        }
+
+        private OsuSpriteText? countText;
+
+        [Resolved]
+        private OverlayColourProvider? colourProvider { get; set; }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Masking = true;
+            Size = new Vector2(20, 12);
+            Alpha = 0f;
+
+            Children = new Drawable[]
+            {
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colourProvider!.Colour1,
```

Comment:
Not sure why this is in the colour provider theme rather than yellow as on design. There's really not enough contrast right now to be able to easily see that there's a channel with mentions, which a bright colour with an entirely distinct hue would help with.